src/server/motoko.ts

```diff
@@ -1,8 +1,8 @@
-// import getMotoko from 'motoko/lib';
-import mo from 'motoko';
+import getMotoko from 'motoko/lib';
+// import mo from 'motoko';
```

Comment:
should this be removed?

Reply:
Correct, these changes would need to be removed before merging this. But they are helpful for local development, so let's keep them in the unmerged PR until Serokell's contributions are merged into moc.

I'll change the PR title to indicate it's not supposed to be merged just yet.

Reply:
Are we ready to merge? I can take over releasing these changes if it would be beneficial for the upcoming demo.

Reply:
> Are we ready to merge?

From my p.o.v., we're ready to merge!